crates/typst-library/src/layout/transform.rs

```diff
@@ -204,20 +198,20 @@ cast! {
     length: Length => ScaleAmount::Length(length),
 }
 
-/// Skews content.
+/// コンテンツのせん断変形。
 ///
-/// Skews an element in horizontal and/or vertical direction. The layout will
-/// act as if the element was not skewed unless you specify `{reflow: true}`.
+/// 水平方向または垂直方向、あるいは両方向に要素をせん断変形します。
+/// `{reflow: true}`を指定しない限り、レイアウトは要素がせん断変形を受けていないかのように振る舞います。
 ///
-/// # Example
+/// # 例
 /// ```example
 /// #skew(ax: -12deg)[
 ///   This is some fake italic text.
 /// ]
 /// ```
 #[elem(Show)]
 pub struct SkewElem {
-    /// The horizontal skewing angle.
+    /// 水平方向のせん断角。
```

Comment:
Skewを「せん断変形」と訳したのはうまいと思いました。 一方で「せん断角」というと機械加工の分野で使われている別の意味の言葉となってしまうため、傾斜角などにしてはいかがでしょうか？
```suggestion
    /// 水平方向の傾斜角。
```